site/content/extend/plugins/webapp/hello-world.md

```diff
@@ -117,15 +117,14 @@ Now, we'll need to define the required manifest describing your plugin's entry p
 ```json
 {
     "id": "com.mattermost.webapp-hello-world",
-    "name": "helloworld",
-    "description": "",
+    "name": "Hello World",
     "webapp": {
         "bundle_path": "main.js"
     }
 }
 ```
 
-This manifest gives the server the location of our web app  within our bundle. (Note that you may alternatively use `plugin.yaml`, as shown in [../../server/hello-world/](../../server/hello-world/).)
+This manifest gives the server the location of our web app  within our bundle. (Note that you may alternatively use `plugin.yaml`) Consult the [manifest reference]({{< ref "manifest-reference.md" >}}) for more details, including how to define a minimum required server version for your plugin.
```

Comment:
Is the first sentence accurate - it gives the location of `the web app` within the plugin bundle? Or is the webapp components of the plugin?

Reply:
If it is, propose tweaking this section to

```
This manifest gives the server the location of our web app within our plugin bundle. Consult the [manifest reference]({{< ref "manifest-reference.md" >}}) for more details, including how to define a minimum required server version for your plugin.

Note that you may also use `plugin.yaml` to define the manifest.
```

Reply:
Fixed